### ISSUE TITLE
[A11Y] Selecting "Editorial" triggers a text element appearing below the section component

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9350]
## Issue Description
// What is the nature of the issue? How does it impact users?

In the "Begin Your Submission" introductory page while selecting the section of the journal in which the submission appears, a user selects some options, e.g.: Editorial then a new text element shows up below Section component. Given this new element relates to the selection, it should be announced to Screenreader users.

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Create a new submission
2. You will encounter the problem on the first page when you click the "New Submission" button while selecting the section in which needs to be published

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4

## Stack used 

- Device: Laptop
- OS: MacOS (Monterey)
- Browser: Chrome
- Version: 117

## Testing method

- Manual (keyword)
- Screen Reader (VoiceOver)

## Action
// Describe action that can be taken to remediate the issue.

Adjust the area/component adding role=status to the element container below Section element. It will announce any modification that is loaded on the area. 

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![image](https://github.com/pkp/pkp-lib/assets/112014397/3b114a0b-88b9-40d3-9cc1-7e75b22ee5cf)
